pkgs/development/python-modules/pikepdf/default.nix

```diff
@@ -0,0 +1,68 @@
+{ attrs
+, buildPythonPackage
+, defusedxml
+, fetchPypi
+, hypothesis
+, isPy3k
+, lxml
+, pillow
+, pybind11
+, pytest-helpers-namespace
+, pytest-timeout
+, pytest_xdist
+, pytestrunner
+, python-xmp-toolkit
+, python3
+, qpdf
+, setuptools_scm
+, setuptools-scm-git-archive
+, stdenv
+}:
+
+buildPythonPackage rec {
+  pname = "pikepdf";
+  version = "1.1.0";
+  disabled = ! isPy3k;
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "14b36r6h3088z2sxp2pqvm171js53hz53mwm1g52iadignjnp0my";
+  };
+
+  buildInputs = [
+    pybind11
+    qpdf
+    setuptools_scm
```

Comment:
`setuptools*` are `nativeBuildInputs`. `pybind` belongs in `buildInputs`, don't know about `qpdf`.

Reply:
Fixed the setuptools* and checked qpdf; it won't work if it's in nativeBuildInputs.